library/build.gradle.kts

```diff
@@ -50,33 +52,44 @@ android {
             suppressWarnings = false
         }
     }
+
+    // Custom ktlint script
+    tasks.register("ktlint") {
+        logger.lifecycle("ktlint")
+        exec {
+            commandLine = listOf("./../scripts/lint.sh", "--no-error")
+        }
+    }
+
+    // Make ktlint run before build
+    tasks.getByName("preBuild").dependsOn("ktlint")
 }
 
 dependencies {
     // Bundled
-    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Constants.BuildScript.kotlinVersion}")
-    implementation("androidx.annotation:annotation:1.5.0")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Constants.BuildScript.kotlinVersion}")
+    implementation("androidx.annotation:annotation:1.6.0")
     implementation("com.google.code.gson:gson:2.8.9")
     implementation("com.jakewharton.threetenabp:threetenabp:1.1.1")
     // DO NOT UPGRADE ABOVE 3.12.X! Version 3.12 is the last version supporting TLS 1 and 1.1
     // If upgraded, the app will crash on android 4.4
     implementation("com.squareup.okhttp3:okhttp:3.12.13")
-    implementation("com.wultra.android.powerauth:powerauth-networking:1.1.3")
+    implementation("com.wultra.android.powerauth:powerauth-networking:1.2.0")
```

Comment:
Where does this come from. The latest version I see is 1.1.3.

Reply:
Oh, it's been released more than 6 months but somehow there is a missing tag (probably a script error), I'll fix that